ops/dev2/_config.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
```

Comment:
Not blocking.

I wonder if, at some point, we could reduce this environment duplication by using the terraform workspace feature and passing in arguments for the `env`, `env_level`, and other variables that differentiate environments (dev, dev2, dev3, pentest). Unfortunately, that may lead to a can of worms, it would probably require a large refactor, and it tends to increase mental load when conceptualizing and performing terraform actions. At this point, I'd struggle to suggest we do this since I don't see us needing to scale out our environment list much more. Still, it has the potential to lower the barrier to creating new envs if it does ever come up. Whenever I see even a somewhat significant amount of duplicated terraform for individual environments, this goes through my mind.

Reply:
We've talked about using workspaces for this before, but it's a really big refactor and we were never sure if it was ultimately worth it at this point.

Reply:
Let's discuss at our next DevOps standup.